src/client/playaudio.js

```diff
@@ -8,9 +8,10 @@
  * @param {Audio} audio - The audio object.
  * @param {float} [volume=1] - Loudness of audio.
  */
-function PLAY_AUDIO(audio, volume=1) {
+
+function PLAY_AUDIO(audio, vol) {
   const concurrentAudio = audio.cloneNode();
-  concurrentAudio.volume = volume;
```

Comment:
Why change volume to vol? It's a good practice to be explicit in what you want to do, and abbreviating names could bad for readability

Reply:
:+1: Also, the default parameter is there for a good reason.